packages/gated-content/package.json

```diff
@@ -116,12 +116,5 @@
       "web/index.ts"
     ],
     "exports": true
-  },
-  "pnpm": {
```

Comment:
pnpm was complaining about this, it should be defined at the root of the monorepo

Reply:
I think the intent was to not have this warning screaming at people using `pnpm` to install these packages.

Reply:
Actually I don't know if this is effective if done on what, from the consumer perspective, is a dependency `package.json`

Reply:
I removed this as pnpm complains, when running install with:

```
packages/gated-content                   |  WARN  The field "pnpm" was found in /lens-sdk/packages/gated-content/package.json. 
This will not take effect. You should configure "pnpm" at the root of the workspace instead.
```